client/src/__tests__/NavBar.test.jsx

```diff
@@ -0,0 +1,24 @@
+import { render, screen } from "@testing-library/react";
+import { BrowserRouter as Router } from 'react-router-dom';
+
+import NavBar from "../components/NavBar";
+
+test('NavBar should render links case-insensitively', () => {
+    render(
+    <Router>
+        <NavBar />
```

Comment:
This is more of a general comment. Do you need to pass down props to this component to get it to work? Think about it!